### PR TITLE
#5268 beakerx jupyterlab extension

### PR DESCRIPTION
**Pull request by MariuszJurowicz**
This PR is for local tests of beakerx jupyterlab extension.
This PR is based on the `0.6.3-widgets7b` branch with the ipywidgets v7 compatible beakerx version.

### CODE REVIEW

File: beakerx/js/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "beakerx",
-  "version": "0.6.3-widgets7b",
+  "version": "0.7.1",
```
Comment (MariuszJurowicz) [inline beakerx/js/package.json]: The version no. is changed due to issues with the `0.6.3-widgets7b` version that can't be used in jupyterlab.
This version should be used to local tests only.

**Comment by MariuszJurowicz** [comment]
At the end i would like to drop the build for the lab extension and use:
```
  "dependencies": {
    "@jupyter-widgets/jupyterlab-manager": "^0.28.0",
    "beakerx": "latest"
  },
```

But now i can't use `"beakerx": "0.6.3-widgets7b"`.
So i have to wait until this version is merged to master branch.

**Comment by MariuszJurowicz** [comment]
@scottdraves 
It requires the https://github.com/twosigma/beakerx/issues/6364 to make the python examples run properly.

**Comment by scottdraves** [comment]
🐬 🐬

**Pull request merged**